### DISCUSSION TITLE
posts/phd-notes/

### DISCUSSION THREAD

**Discussion by giscus** [brant-ruan/fernweh-comments#11] [Announcements]
# posts/phd-notes/

读博小悟（长期更新）

https://blog.wohin.me/posts/phd-notes/

**Comment by ListenerMoya** [comment]
追更

**Reply by brant-ruan** [reply]
Stay tuned :)

**Comment by zzzhitong** [comment]
您好！我是从google搜索关键字看到这篇博客的，读完受益良多；再往前一翻，发现博主竟也是同济信安的学长，前几年曾在知乎上见过您的回答，不免觉得神奇。我是24届信安专业的毕业生，现在正在读和安全不太相关的方向的研，对很多东西产生了很多困惑；但读了几篇您的博文后，意外地感到有些重拾了曾经有过的热忱和探求之心。不知这样说是否合适，但很感谢学长的文字，以一种很神奇的方式链接了我的两个生活节点，在不同的时期一定程度上帮助了我。祝您生活顺利！

**Reply by brant-ruan** [reply]
你好👋，能够给你带来一些触动，我也很开心～也祝你生活顺利，学业有成！
